test/pooling.jl

```diff
@@ -248,6 +248,70 @@ meanpool_answer_dict = Dict(
     )
 )
 
+lppool_answer_dict = Dict(
```

Comment:
where did you take these values? pytorch? or "manual"computation?

Reply:
Part from pytorch `LPPool1d` and `LPPool2d` (normal & stride), and part from calculator (pad, because pytorch `LPPool` doesn't support pad)